test/e2e/federated-service.go

```diff
@@ -275,7 +266,8 @@ func createService(fcs *federation_release_1_3.Clientset, clusterClientSets []*r
 		},
 	}
 	nservice, err := fcs.Core().Services(namespace).Create(service)
-	Expect(err).NotTo(HaveOccurred())
+	framework.Logf("Trying to create service %q in namespace %q", service.ObjectMeta.Name, service.ObjectMeta.Namespace)
+	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("creating service %s: %+v", service.Name, err))
```

Comment:
The log messages are also output when the --verbose option is used, even when they pass.  I'd suggest we leave as is.